src/west/_bootstrap/main.py

```diff
@@ -167,7 +166,9 @@ def init(argv):
         west_dir(args.directory)
         sys.exit('West already initialized. Aborting.')
     except WestNotFound:
-        bootstrap(args)
+        pass
```

Comment:
Ah, okay... I see you removed the exception cleanup in the previous patch only to replace it in this patch.

This series has a couple places where it's hard to read in patch order or the patches are not bisect friendly. I won't ask for changes to fix this, again, since we're in a hurry, but I think in general it's easier to review if the patches can be read in order (and of course not breaking bisect is a good thing).

Reply:
I agree with you.

Guess the first change could have been done more carefully :(
I tried to be careful, especially on the Update -> SelfUpdate and then introduction of the new update command.
But not careful enough.
(would have like this as a couple independent PRs as well, but as you mention, hurry)